docs/user/dashboard/edit-dashboards.asciidoc

```diff
@@ -81,6 +81,20 @@ Put the dashboard in *Edit* mode, then use the following options:
 * To delete, open the panel menu, then select *Delete from dashboard*. When you delete a panel from the dashboard, the 
 visualization or saved search from the panel is still available in Kibana.
 
+[float]
+[[sync-colors]]
+=== Synchronize colors
+
+By default, Kibana attempts to keep colors for series consistent across panels, making sure the same series name always uses the same color.
```

Comment:
Glad you've written docs! I think what you've written is helpful, but I would also like to add answers to these two questions:

1. What are the limitations on color syncing? I know that one of the limitations is that they need to share the same palette.
2. What causes the "too many colors" issue?

```suggestion
By default, dashboard panels that share a color palette will synchronize their color assignment to improve readability. Color assignment is based on the series name, and the total number of colors is based on the number of unique series names.
```